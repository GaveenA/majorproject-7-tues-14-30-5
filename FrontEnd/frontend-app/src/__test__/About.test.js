import React from 'react';
import ReactDOM from 'react-dom';
import {render} from '@testing-library/react';
import About from '../components/Home/About';
import renderer from 'react-test-renderer';


it("renders About button without crashing",() => {
    const dev = document.createElement("dev");
    ReactDOM.render(<About></About>, dev);
})


it("About Class Snapshot is Matched",() => {
  
    const tree = renderer.create(<About></About>).toJSON();
    expect(tree).toMatchSnapshot();
})