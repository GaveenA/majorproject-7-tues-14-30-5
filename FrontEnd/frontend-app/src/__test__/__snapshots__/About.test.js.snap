// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Class Snapshot is Matched 1`] = `
<div>
  <h1>
    ABOUT PAGE HERE
  </h1>
</div>
`;
